### PR TITLE
8.2 update to 1.1.1k-9

### PR DESCRIPTION
**Pull request by bleader**
Merge branch 'upstream' into 8.2-update-to-1.1.1k-9

- Update to 1.1.1k-9 from CentOS 8 Stream
- Renamed Patch100 to Patch500 as original package now has Patch100
- Removed Patch1000 (update-expired-SCT-certificates.backport) as it is now integrated upstream

### CODE REVIEW

File: SPECS/xs-openssl.spec [hunk 1]
```diff
@@ -24,7 +24,7 @@
 Summary: Utilities from the general purpose cryptography library with TLS implementation
 Name: xs-openssl
 Version: 1.1.1k
-Release: 6.1%{?dist}
+Release: 9%{?dist}
```
Comment (stormi) [inline SPECS/xs-openssl.spec]: Should be:
```suggestion
Release: 9.1%{?dist}
```

Comment (bleader) [reply SPECS/xs-openssl.spec]: I was unsure if we wanted to add the .1 for the first version, but I forgot to remove it from the changelog so it wasn't even coherent... Fixed.

Comment (stormi) [reply SPECS/xs-openssl.spec]: The rationale here is that -9 is the upstream release, so we leave it to them and go with -9.1

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
Koji build (v8.2-incoming): http://koji.xcp-ng.org/taskinfo?taskID=40363

**Pull request merged**